tests/functional/glusterd/test_glusterd_start_stop_output.py

```diff
@@ -44,26 +56,25 @@ def test_glusterd_start_stop_consistent_info(self):
         2. Check if consistent output is returned if glusterd is started and
         stopped 5 times.
         """
-        glusterd_start_cmd = 'systemctl start glusterd'
-        glusterd_stop_cmd = 'systemctl stop glusterd'
-
         # Stop glusterd to get the info for further comparison.
-        stop_output = self._run_command_in_servers(glusterd_stop_cmd)
+        stop_output = self._run_command_in_servers(self.glusterd_stop_cmd)
 
         # Start glusterd to get the info for further comparison.
-        start_output = self._run_command_in_servers(glusterd_start_cmd)
+        start_output = self._run_command_in_servers(self.glusterd_start_cmd)
 
         # Loop over the stop-stop cycle for 5 times and validate the output.
         for _ in range(5):
```

Comment:
I feel we should have a call to reset-failed before the for loop. From setup, we've already run start glusterd for 2 times. And If I remember it right, the burst limit is 6 within an hour.

Reply:
ack, i can just remove starting glusterd explicitly in setup and that'd be find as we'll not be breaching 6.

```
StartLimitBurst=6
StartLimitIntervalSec=3600
StartLimitInterval=3600
```